library/core/src/time.rs

```diff
@@ -1232,7 +1232,7 @@ impl fmt::Debug for Duration {
                 self.nanos.0 % NANOS_PER_MICRO,
                 NANOS_PER_MICRO / 10,
                 prefix,
-                "µs",
+                "μs",
```

Comment:
Could you add a comment here mentioning that there are two mu's in Unicode, but that this one is preferred?